examples/src/main/java/io/kubernetes/client/examples/KubectlJsonExample.java

```diff
@@ -0,0 +1,46 @@
+package io.kubernetes.client.examples;
+
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
+import com.squareup.okhttp.Response;
+import io.kubernetes.client.ApiClient;
+import io.kubernetes.client.ApiException;
+import io.kubernetes.client.Configuration;
+import io.kubernetes.client.apis.CoreV1Api;
+import io.kubernetes.client.util.Config;
+import java.io.IOException;
+
+/**
+ * A simple example of how to use the Java API
+ *
+ * <p>Easiest way to run this: mvn exec:java
+ * -Dexec.mainClass="io.kubernetes.client.examples.CopyExample"
+ *
+ * <p>From inside $REPO_DIR/examples
+ *
+ * <p>mock "kubectl get configamp configmap-name -o json"
```

Comment:
what's the point of adding this example? i don't think this example worth a new file..

Reply:
hi @yue9944882
I use java-client to work for pass platform, I need to get xxx.json to change, mock kubectl get resourcename -o json。but Under the existing examples，I don't konw how to do it. Stand in the user's Perspective ，I add example。

It's up to you to decide.

Best Wishes